torch/onnx/experimental/fonnx/v1/custom_operators.py

```diff
@@ -0,0 +1,63 @@
+from contextlib import contextmanager
+
+
+def register_custom_operator(
+    domain: str,
```

Comment:
nit: 'domain' and 'operator' is in the past. It should be the new aten identifier format.

Reply:
How would that API be? Something like `def register_custom_operator(operator: torch._ops.OpOverload, opset_version: int, symbolic_fn: Callable)`? In this approach, how would a user register a custom "com.microsoft::MagicLayerNorm"?

The original proposal was trying to hide the internal (FX) implementation because we didn't establish an "Export graph ir" or "onnxscript ir". I am open to a different approach, let's discuss this is more detail

Reply:
`Union[Callable, torch._ops.OpOverload]` for v1 per discussion